src/key_schedule.cpp

```diff
@@ -554,11 +581,26 @@ KeyScheduleEpoch::do_export(const std::string& label,
   return suite.expand_with_label(secret, "exporter", context_hash, size);
 }
 
+PSKWithSecret
+KeyScheduleEpoch::branch_psk(const bytes& group_id, epoch_t epoch)
+{
+  auto nonce = random_bytes(suite.secret_size());
+  return { { BranchPSK{ group_id, epoch }, nonce }, resumption_secret };
+}
```

Comment:
where is ```BranchPSK``` defined ? Am I not seeing it ?

Reply:
`include/mls/messages.h`